Makefile

```diff
@@ -11,6 +11,10 @@ build:
 test: build/universal-robots
 	./build/universal-robots-test
 
+tidy: build/universal-robots
```

Comment:
was torn between `make tidy` and `make bless`. seemed a bit unnecessary to make them make commands